etc/config.xml

```diff
@@ -29,7 +29,7 @@
                 <house_number_street_line>0</house_number_street_line>
                 <webhook_notification_processor>cron</webhook_notification_processor>
                 <adyen_support_email_address>support@adyen.com</adyen_support_email_address>
-                <payment_pre_authorized>processing</payment_pre_authorized>
+                <payment_pre_authorized>adyen_authorized</payment_pre_authorized>
```

Comment:
Do we need migration for that change?

Reply:
We have a data patch in the PR for the merchants who have already upgraded to 9.0.2 to revert the usage of `processing` state. If the merchant doesn't have `processing` state set in the configuration, we don't need to patch the data assuming that they have the correct configuration for `payment_pre_authorized` field.